timeusage/views.py

```diff
@@ -0,0 +1,80 @@
+from django.views.generic import ListView
+from feedback.views import ManageCourseMixin, update_context_for_feedbacks
+from feedback.models import Feedback
+from feedback.filters import FeedbackFilter
+
+from plotly.offline import plot
+from plotly.graph_objs import Bar, Figure, Layout
+
+
+class TimeUsageView(ManageCourseMixin, ListView):
+
+    model = Feedback
+    template_name = "time_usage.html"
+
+    def plot_times(self):
+        feedbacks = self.get_queryset()
+        feedbacks_by_exercise = {}
+        for f in feedbacks:
+            feedbacks_by_exercise.setdefault(f.exercise, []).append(f)
+        
+        x_data = []
```

Comment:
I'm pretty sure this can be written prettier, but that is _not_ important of course.